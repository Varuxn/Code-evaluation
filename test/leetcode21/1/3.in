100 100
lkypnlzdcekvidytcqpfrowfggsgqaubmsrb
seiwqfeudawvpbnfzulhmtxonb
fscjkjtizaptdprbgvoluiyewrdkfelkyqvkzqty
kuwzlxhgnudxshtlkdsrpfphcagtvgl
ahgmgnutjx
dejornjkeozlqbtlwzzdcgjonykihiz
oibhxkrbytoquhetjdyllibailjsvkekufrsrlvrgki
toverwpeesfoeqgball
sfokskb
wlarzywsunxaieoowxpyibucijmav
yrzakbyituwqwewns
khquklw
wlurzsiavubvdvqznofmbzgnizhtmgnirizscjszdwvgtnh
boufndswcaroigwbqxtvgnuljpuee
ngqjlfmddofwcpfaqvxjsfznriel
jocrglemajrorotgtvxqtijahyrscehl
lezyinysfpjtkpofoeazpbgn
aqzjduuittsjtkoivju
xabdcatdjgalyzvetrmo
gxgrnqmwlzvndbpfuvqcvcdwzhpqvfdbflusbhroiobnptv
qlqlptjocafxfkymvtfy
ypmmqcelxpbihoyaxmextbf
ar
vydxysllknrwkizvqotsnhxndysia
nvkqukiixsxooixqdpeyhtfggifathogfawbkeliyjwnrv
wkjwsedanofnhpdpwd
xorjwpsuelqjjaufubivowcdfrhunkhnaa
w
qrwdjimjerefbbvadzfxiamshzvkxrcpkystief
xjub
qbprgoz
btqcoacgcrscmonqtcqeyrqqshhyw
hzazdob
wfzpjlfwdyavczmupgbzhzfqyhpcws
uxklgvqfbofwqelkwrovyncookfsixbcumodjeiktnjluuysno
obrepbkhjjlmezcjigrvbg
naanspbhdebiucsdodqchpesamjxoqd
sfqnwtwa
xitaczqfrspjyj
ktiylnbdutsqopsomchogieoawdj
ouhhefurjanesfthyjwaskhbpvdnamkoirymwsfhu
mnx
wvqsyidfjucmhcarskiqwhldofva
vhtszlqiqxrmzgudinwtymrfxvvfsyz
fsifgapyygnzojdzwz
unl
ngxuzvtocowjwxywxhjwxsbwrconqapfio
hluypkuzgtzfqgqngj
ccrtpjvgqdu
nhjlxuhycbxjrgagml
qnqjezelpkhgxppkolrmqu
zotb
hmtybjhgkotzychxtzhhk
vcvhelcfhkubixkpfvdywdbeaudjepk
uhiyslfzvzcfzpuekadifgogctpglbifisfcdkeamghlvcpheu
lagtfclnnmvuxozzt
fwvodgjhytnslbvtuavfnksl
rzdsupyskmytxhrqvldwgxsguyu
sfljgrbcgzxsnvnlcgdzshxygqfaobhgiv
qmsssspmhmaspgwqydozleel
hvcrqthelbwftinfihwrdnp
dothuhddcfuuydylfwtyggfrncitrzduqxd
egqinmelqeyvarvizczpehixjlrzkwlqc
aroecekdbmuyvwaulhevgpiaotwzjbdmsrqxxdaa
wyksah
jnaqcjqs
ornpuzknshkvjmk
mxapgfytiqxrgpvxiiofky
chfzqtlygiavqhv
pmiguzdrkmgpwgjaoq
gkkesvgnlqixfwhorjri
fqkdxvdlmc
wpyqkgdxwnwekgsdplnktewwdrcqdehbvhrhowfmkb
wjk
ayokruinxapnfwxhtgabuxgjjjzhvmivkw
brqqqqgewedfxjfageippuowpaekk
sn
lembmszqvgpfoqlsyakuqgjtnufsotozzcdlved
lsxbkivlkffdmpwbldtbyhbxjgi
klxxguyqfubpzjslyqoluhmsopqavyegmddsycifymuxv
lvfzgzhuuvkkxfkenwhtqhxzmvnjsi
dfddnfmjzjtlhbvlqusjkcgloby
uivznbeciqmhchsjkpwb
okuqtfhwggqoesefygnruw
cqcmfynpoalhvq
rwwknac
hazwruumwkokqoahfcsoxieohayudclmfklygfleqbpi
ppxrhlqrreytd
wfhlkuwjadwftxwdpnsmebzutqark
oilvvxprisxexsetwtgoim
jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgk
yhsryriwghjoqfiajfzfxwccnzpr
cggjyxhrifyppofuygfehmddohsnxlvzsbksbrkjzka
ygmyorcxffctp
imfenzhzsisctrmuhmc
kqmpnrwpml
vzqboqinkt
mmzaijo
eaatrpmoezbpwdixrafhkapynqyvbogioibiypweo
wmcejugrdqruogmocockys
csqyxfavtcbdyhwdxnxnwkbaafkyzezczpcywetqhuing hluypkuzgtzfqgqngjing jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgkes ocuvoswtryshdqfhlvqfqyghlgqvtsqjxmglhcgadajispping idiqmtycotxgeuvkhcpoesqocjftmpbxujpheed mmzaijoed yamcpahjqxlbchorinoucvysgtcpxqtvsfykhing jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgked egqinmelqeyvarvizczpehixjlrzkwlqced uhiyslfzvzcfzpuekadifgogctpglbifisfcdkeamghlvcpheued irzrkbgjtzing fymmtmqcoyqfxrknjrxces yrudfbkfnwzcobqmuxmlhzuawztlophmidqoeatuzuwing cqlqdvpzvnyoymfizolreqtxmuskkqamilclgtmes zapaqarusntghnqrdqing wyksahing clqjylsxlhabjrxegqkes toverwpeesfoeqgballing dejornjkeozlqbtlwzzdcgjonykihizing mbiemgpegclgfwzcagtlwzqdijsdlrbxsjdestkaing hvcrqthelbwftinfihwrdnping btqcoacgcrscmonqtcqeyrqqshhywing ahnbxtkilnvequwaycsrmuojcrflzzdbiqdhjnrxcnbthaing qbprgozing jnclgjed jssobddtaficapevxikvbtiggrpfyagjuaxyeatgibkksoed fwvodgjhytnslbvtuavfnksling vgwoybnak ynxiqhtjybavvnxjjeging mevlaijxahmxqding wpyqkgdxwnwekgsdplnktewwdrcqdehbvhrhowfmkbing cing tnfbmlkbpimsqxpqebqwezhsevhidjjwyrxkcjing zmlwdcdguzntjfzeojhzsfxleiqqvkjved qmsssspmhmaspgwqydozleeling vwuqing jsarsuyrgdbwing lezyinysfpjtkpofoeazpbgnqwerqwe boufndswcaroigwbqxtvgnuljpueeqwerqwe nhreqjejdzivhnydivnontqbcuhuuljhuaokkuwping atlfdgsrwnjxpeges mmzaijoing xisvxlbmowxvzdqtbdeavigjizesmcpkmhhmvjylhvijayceqwerqwe obrepbkhjjlmezcjigrvbged tztllfusrpdpclmluing egqinmelqeyvarvizczpehixjlrzkwlqcing jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgking kcbztwyojltednvheyutjoiuobmwaewlhzmcxkqhxjlayijdgded yrzakbyituwqwewnsed oibhxkrbytoquhetjdyllibailjsvkekufrsrlvrgkied upbgrashhptusrdbwjhudaaing toing ouhhefurjanesfthyjwaskhbpvdnamkoirymwsfhuing mnxes dijzaccdbokvrlykvhdlaefqvaoevewymhyed mqckjpitmryzudekaeygaqwerqwe nvkqukiixsxooixqdpeyhtfggifathogfawbkeliyjwnrving wlarzywsunxaieoowxpyibucijmaving brqqqqgewedfxjfageippuowpaekked wing txjncoxcuxjublxsxxasabbwfing mnxed dothuhddcfuuydylfwtyggfrncitrzduqxd ffaobcbawayysaauyhpbkldvbbsfes aqzjduuittsjtkoivjuing fqkdxvdlmcing kdpivskrqsiscuing ysceylnkvmzgpqssgbing jofriinhikicnmayamvmledced linwysnjabes alrbndying sning sfqnwtwaed nnjngzwicnvpnjsyatlging wfzpjlfwdyavczmupgbzhzfqyhpcwsqwerqwe xmmlxzrwwbylwqyiixkdkqwznsvfpsepeqed amwaxtqxbyulbheyirsezkydzcwbfmwhased nvkqukiixsxooixqdpeyhtfggifathogfawbkeliyjwnrves wvqsyidfjucmhcarskiqwhldofva xqahogiscjaohhqzrpnsnfrgjviryvtvlteczmubxwing jfvyupoaiwjefdecwbprfrttowqkowoxbmyed mijqpowtkxjecaxvqtning rzdsupyskmytxhrqvldwgxsguyuing qrwdjimjerefbbvadzfxiamshzvkxrcpkystiefing vzqboqinkting wyksahing uivznbeciqmhchsjkpwbing ayokruinxapnfwxhtgabuxgjjjzhvmivkwing gbfkechuxuvhfned gdyuydulchwvicwnoarvvqrsxwfr jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgkqwerqwe hzazdobes sxhuocpeuiespvmpqmvgjxcaoing yivqfcling bckhvcdjrvwmbhlghcjjghring hlvonwqaencjxvevkhcxnlzyuiinlinttihiezikped zjuing obrepbkhjjlmezcjigrvbges hzazdobing aqzjduuittsjtkoivjued
