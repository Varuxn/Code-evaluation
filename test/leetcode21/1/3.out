csqyxfavtcbdyhwdxnxnwkbaafkyzezczpcywetqhuing hluypkuzgtzfqgqngj jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgk ocuvoswtryshdqfhlvqfqyghlgqvtsqjxmglhcgadajispping idiqmtycotxgeuvkhcpoesqocjftmpbxujpheed mmzaijo yamcpahjqxlbchorinoucvysgtcpxqtvsfykhing jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgk egqinmelqeyvarvizczpehixjlrzkwlqc uhiyslfzvzcfzpuekadifgogctpglbifisfcdkeamghlvcpheu irzrkbgjtzing fymmtmqcoyqfxrknjrxces yrudfbkfnwzcobqmuxmlhzuawztlophmidqoeatuzuwing cqlqdvpzvnyoymfizolreqtxmuskkqamilclgtmes zapaqarusntghnqrdqing w clqjylsxlhabjrxegqkes toverwpeesfoeqgball dejornjkeozlqbtlwzzdcgjonykihiz mbiemgpegclgfwzcagtlwzqdijsdlrbxsjdestkaing hvcrqthelbwftinfihwrdnp btqcoacgcrscmonqtcqeyrqqshhyw ahnbxtkilnvequwaycsrmuojcrflzzdbiqdhjnrxcnbthaing qbprgoz jnclgjed jssobddtaficapevxikvbtiggrpfyagjuaxyeatgibkksoed fwvodgjhytnslbvtuavfnksl vgwoybnak ynxiqhtjybavvnxjjeging mevlaijxahmxqding w cing tnfbmlkbpimsqxpqebqwezhsevhidjjwyrxkcjing zmlwdcdguzntjfzeojhzsfxleiqqvkjved qmsssspmhmaspgwqydozleel vwuqing jsarsuyrgdbwing lezyinysfpjtkpofoeazpbgn boufndswcaroigwbqxtvgnuljpuee nhreqjejdzivhnydivnontqbcuhuuljhuaokkuwping atlfdgsrwnjxpeges mmzaijo xisvxlbmowxvzdqtbdeavigjizesmcpkmhhmvjylhvijayceqwerqwe obrepbkhjjlmezcjigrvbg tztllfusrpdpclmluing egqinmelqeyvarvizczpehixjlrzkwlqc jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgk kcbztwyojltednvheyutjoiuobmwaewlhzmcxkqhxjlayijdgded yrzakbyituwqwewns oibhxkrbytoquhetjdyllibailjsvkekufrsrlvrgki upbgrashhptusrdbwjhudaaing toing ouhhefurjanesfthyjwaskhbpvdnamkoirymwsfhu mnx dijzaccdbokvrlykvhdlaefqvaoevewymhyed mqckjpitmryzudekaeygaqwerqwe nvkqukiixsxooixqdpeyhtfggifathogfawbkeliyjwnrv w brqqqqgewedfxjfageippuowpaekk w txjncoxcuxjublxsxxasabbwfing mnx dothuhddcfuuydylfwtyggfrncitrzduqxd ffaobcbawayysaauyhpbkldvbbsfes aqzjduuittsjtkoivju fqkdxvdlmc kdpivskrqsiscuing ysceylnkvmzgpqssgbing jofriinhikicnmayamvmledced linwysnjabes alrbndying sn sfqnwtwa nnjngzwicnvpnjsyatlging w xmmlxzrwwbylwqyiixkdkqwznsvfpsepeqed amwaxtqxbyulbheyirsezkydzcwbfmwhased nvkqukiixsxooixqdpeyhtfggifathogfawbkeliyjwnrv w xqahogiscjaohhqzrpnsnfrgjviryvtvlteczmubxwing jfvyupoaiwjefdecwbprfrttowqkowoxbmyed mijqpowtkxjecaxvqtning rzdsupyskmytxhrqvldwgxsguyu qrwdjimjerefbbvadzfxiamshzvkxrcpkystief vzqboqinkt w uivznbeciqmhchsjkpwb ayokruinxapnfwxhtgabuxgjjjzhvmivkw gbfkechuxuvhfned gdyuydulchwvicwnoarvvqrsxwfr jilciemoswbouafljxkpwdtpzpxpxlhitskexyupvvfrvlcgk hzazdob sxhuocpeuiespvmpqmvgjxcaoing yivqfcling bckhvcdjrvwmbhlghcjjghring hlvonwqaencjxvevkhcxnlzyuiinlinttihiezikped zjuing obrepbkhjjlmezcjigrvbg hzazdob aqzjduuittsjtkoivju